internal-functions

```diff
@@ -32,6 +32,8 @@ cmd-scheduler-kubernetes-rolling-update() {
   if [[ -n "$NAMESPACE" ]]; then
     KUBE_ARGS+=("--namespace=$NAMESPACE")
     fn-scheduler-kubernetes-ensure-namespace "$NAMESPACE" >/dev/null
+  else
+    KUBE_ARGS+=("--namespace=default")
```

Comment:
Maybe change this:

```
NAMESPACE="$(fn-plugin-property-get "scheduler-kubernetes" "$APP" "namespace" "")"
```

to this:

```
NAMESPACE="$(fn-plugin-property-get "scheduler-kubernetes" "$APP" "namespace" "default")"
```

and have `fn-scheduler-kubernetes-ensure-namespace` return immediately if the `NAMESPACE` argument is `default`? That would minimize the code here.

Reply:
If i understood you correctly, the above becomes something like this
```
NAMESPACE="$(fn-plugin-property-get "scheduler-kubernetes" "$APP" "namespace" "default")"
KUBE_ARGS+=("--namespace=$NAMESPACE")
fn-scheduler-kubernetes-ensure-namespace "$NAMESPACE" >/dev/null
```

and in `fn-scheduler-kubernetes-ensure-namespace` we have
```
...
if [[  "$NAMESPACE" = "default" ]]; then
    return
fi
SIGIL_PARAMS=(NAME="$NAMESPACE")
...
```

Reply:
yeah more or less :)

Reply:
changes effected